lib/mutant/isolation.rb

```diff
@@ -33,30 +37,41 @@ module Fork
       #   if block terminates abnormal
       #
       # rubocop:disable MethodLength
-      def self.call(&block)
-        IO.pipe(binmode: true) do |reader, writer|
-          writer.binmode
+      def call(&block)
+        io.pipe(binmode: true) do |reader, writer|
           begin
```

Comment:
I dunno if I would go that far, since I can't see such an object being anything more than a container.